proto/spire/server/hostservices/README_pb.md

```diff
@@ -3,63 +3,66 @@
 
 ## Table of Contents
 
-- [identityprovider.proto](#identityprovider.proto)
-    - [FetchX509IdentityRequest](#spire.server.hostservices.FetchX509IdentityRequest)
-    - [FetchX509IdentityResponse](#spire.server.hostservices.FetchX509IdentityResponse)
-    - [X509Identity](#spire.server.hostservices.X509Identity)
+- [agentstore.proto](#agentstore.proto)
```

Comment:
This seems to be replacing the identity provider documentation, is that intentional?

Reply:
Yeah that's weird...

Reply:
Oh yep. The goo in build.sh in charge of generating the README does not handle more than one .proto in a directory and ends up overwriting. I'll fix that real quick.